dts/bindings/ethernet/renesas,ra-ethernet.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas RA Ethernet
+
+compatible: "renesas,ra-ethernet"
+
+include: [ethernet-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  phy-type:
```

Comment:
This is too similar to the generic phy-connection-type. Also I don't think it's a good idea to have an enum of a bunch of different specific phy devices. The phy should be a separate node in DT if it is a separate device in hardware.

Reply:
@decsny , Changed to phy-connection-type